### PR TITLE
Pull release merges into master

### PR DESCRIPTION
**Pull request by mwestphal**
Merge #147 and #141 in master

### CODE REVIEW

File: resources/com.f3d-app.F3D.metainfo.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-    <id>com.kitware.F3D</id>
+    <id>com.f3d-app.F3D</id>
```
Comment (mwestphal) [inline resources/com.f3d-app.F3D.metainfo.xml:3]: @Meakk : this should already be in master ? not sure what to make of that.

Comment (Meakk) [reply resources/com.f3d-app.F3D.metainfo.xml:3]: Yes, I'm not sure why it appears here.

Comment (Meakk) [reply resources/com.f3d-app.F3D.metainfo.xml:3]: Try to merge it and we'll see.

**Review by Meakk — APPROVED** [review]

File: resources/com.f3d-app.F3D.metainfo.xml (hunk 1 above)
Comment (mwestphal) [reply resources/com.f3d-app.F3D.metainfo.xml:3]: alright

Comment (mwestphal) [reply resources/com.f3d-app.F3D.metainfo.xml:3]: commit is visible two times in the historic, but file is perfectly fine. Well, lets not forget to merge into release when needed :)

**Pull request merged**